yt_dlp/extractor/oftv.py

```diff
@@ -0,0 +1,50 @@
+from .common import InfoExtractor
+from .zype import ZypeIE
+from ..utils import traverse_obj
+
+
+class OfTVIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?of.tv/video/(?P<id>[0-9a-zA-Z]+)'
+    _TESTS = [{
+        'url': 'https://of.tv/video/627d7d95b353db0001dadd1a',
+        'md5': 'cb9cd5db3bb9ee0d32bfd7e373d6ef0a',
+        'info_dict': {
+            'id': '627d7d95b353db0001dadd1a',
+            'ext': 'mp4',
+            'title': 'E1: Jacky vs Eric',
+            'thumbnail': r're:^https?://.*\.jpg',
+            'average_rating': 0,
+            'description': 'Singer Jacky Romero and actor Eric Guilmette must step out of their comfort zones in order to take on the ultimate comfort food: the Monte Cristo sandwich. With only 30 minutes to deliver the perfect blend of sweet and savory,\xa0Jacky and Eric need all the help they can get to impress Chef JoJo and move on to the next round.\xa0\r\n\r\nSubscribe to the cast on OF:\r\nof.com/jackyromero \r\nof.com/eric_g \r\nof.com/grandmasterchefjojo',
```

Comment:
use md5

Reply:
was this commit what you were looking for? e7d2ee9063ca06b9b1ea054dc5e7c2042c82c72e

Reply:
for test fields where the string value is long/unwieldy you can use an md5 hash in place of the string
```py
            'description': 'md5:TODO',
```
run the test with this and the result will give you a hash to replace `TODO` with

Reply:
completed in 0e46566531b75a8650f955075b65fb72b0a15f25